docs/network-flow-visibility.md

```diff
@@ -622,3 +633,150 @@ With filters applied:
 
 <img src="https://downloads.antrea.io/static/03022021/flow-visualization-np-2.png" width="900" alt="Flow
 Visualization Network Policy Dashboard">
+
+## Clickhouse-Grafana Flow-visibility Solution
+### Purpose
+
+Antrea supports sending IPFIX flow records through the Flow Exporter feature described 
+above. The Clickhouse-Grafana Flow-visibility Solution works as the data storage and 
+visualization tool for flow records and flow-related information. This document 
+provides the guidelines for deploying Clickhouse-Grafana Flow-visibility Solution 
+with support for Antrea-specific IPFIX fields in a Kubernetes cluster.
+
+### About Clickhouse-Grafana
+
+[Grafana](https://grafana.com/grafana/) is an open-source platform for monitoring 
+and observability. Grafana allows you to query, visualize, alert on and understand 
+your metrics. [Clickhouse](https://clickhouse.com/) is an open-source, high performance 
```

Comment:
high performance -> high-performance

Reply:
I quoted the sentence from https://clickhouse.com/ . I suppose it's better to keep it in-consistent with their wordings?

Reply:
ok, sure.